core/management/commands/data_intake.py

```diff
@@ -0,0 +1,75 @@
+import json
+import pandas as pd
+from django.core.management.base import BaseCommand, CommandParser
+from django.utils import timezone
+from core.models import Person, DataSource, Record
+
+
+class Command(BaseCommand):
+    help = "populates the database with data from a csv file"
+
+    def add_arguments(self, parser: CommandParser) -> None:
+        parser.add_argument("files", nargs="+", type=str)
+
+    def update_db(self, file, cols_as_expected) -> None:
+        data = pd.read_csv(file)
+        filename = file.split("\\")[-1].split(".")[0]
+
+        data.rename(columns=cols_as_expected, inplace=True)
+
+        # separate record data from person data
+        person_cols = [
+            "last_name",
+            "first_name",
+            "date_of_birth",
+            "address",
+            "age",
+            "nhs_number",
+        ]
+
+        # store non-id data as json.        
+        record_df = data[data.columns.difference(person_cols)]
+        json_records = [json.dumps(row) for row in record_df.to_dict("records")]
+
+
+        datasource = DataSource.objects.create(
+            name=filename,
```

Comment:
This won't be the case likely where this data comes from, but since the client hasn't yet defined how this would be sent to us, it's fine for now.